src/algebra/module.lean

```diff
@@ -810,23 +810,25 @@ def add_monoid_hom.to_rat_linear_map [add_comm_group M] [vector_space ℚ M]
   M →ₗ[ℚ] M₂ :=
 ⟨f, f.map_add, f.map_rat_module_smul⟩
 
-namespace finset
+-- FIXME why are these even here? use •ℕ?
 
-variable (R)
+-- namespace finset
 
-lemma sum_const' [semiring R] [add_comm_monoid M] [semimodule R M] {s : finset ι} (b : M) :
-  (∑ i in s, b) = (finset.card s : R) • b :=
-by rw [finset.sum_const, ← semimodule.smul_eq_smul]; refl
+-- variable (R)
 
-variables {R} [decidable_linear_ordered_cancel_add_comm_monoid M]
-  {s : finset ι} (f : ι → M)
+-- lemma sum_const' [semiring R] [add_comm_monoid M] [semimodule R M] {s : finset ι} (b : M) :
```

Comment:
This is used in `archive/sensitivity.lean`: https://github.com/leanprover-community/mathlib/pull/3495/checks?check_run_id=894831487#step:5:55

Reply:
I changed `sensitivity.lean` to just use `sum_const` and `nsmul_eq_mul`.